spec.bs

```diff
@@ -3162,11 +3515,16 @@ An interest group is a [=struct=] with the following [=struct/items=]:
 :: Null or a [=string=]. Additional metadata that the owner can use during on-device bidding.
 : <dfn>ads</dfn>
 :: Null or a [=list=] of [=interest group ad=]. Contains various ads that the interest group might
-  show.
+  show. Must be null if [=interest group/additional bid key=] is not null.

```

Comment:
Related to the above comment: Could you please add `Must be null if [=interest group/additional bid key=] is not null`. to `update url` above?